core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.ts

```diff
@@ -0,0 +1,377 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
```

Comment:
Can we do something such that it only shows the change in the filename? (This will help us to keep the history of this file?)

Reply:
I'm not sure about this @DubeySandeep, sometimes it keeps the file's history but sometimes it doesn't. I don't know what I should do to keep it.